src/AzureExtension/DevBox/Helpers/WingetConfigWrapper.cs

```diff
@@ -328,21 +321,56 @@ private void SetStateForCustomizationTask(TaskJSONToCSClasses.BaseClass response
         }
     }
 
+    // Start the Dev Box if it isn't already running
+    private async Task HandleNonRunningState()
+    {
+        // Check if the dev box might have been started in the meantime
+        var stateResult = await _devBox.GetStateAsync();

```

Comment:
Changed but does it make a difference?

Reply:
It was mostly because the pattern you established below is using the GetState() pattern so my suggestion was more around keeping it consistent. That plus one less async call.